.github/workflows/ci.yml

```diff
@@ -152,12 +152,8 @@ jobs:
       - name: Install wheels
         run: pip install deepgnn_ge*.whl deepgnn_torch*.whl
       - name: Run gat examples
-        run: cd ./pytorch/gat/ && sudo chmod +x *.sh && bash ./run.sh CPU
-      - name: Run pytorch geometric examples
-        run: cd ./pytorch/geometric/gat/ && sudo chmod +x *.sh && bash ./run.sh CPU
+        run: cd .. && bazel test -c dbg --config=linux //examples/pytorch/gat:gat_example
```

Comment:
1. `cd ..` is not needed. 
2. You are not testing wheels with Bazel, but existing code because you are taking dependency on graph engine code in build files(hint there should be no `-c dbg` in a bazel test command). The goal of e2e tests is to verify examples code will work on end user compute instance without our source.